rotkehlchen/exchanges/kraken.py

```diff
@@ -350,13 +351,13 @@ def _query_private(self, method: str, req: dict | None = None) -> dict | str:
         # any unicode strings must be turned to bytes
         hashable = (str(req['nonce']) + post_data).encode()
         message = urlpath.encode() + hashlib.sha256(hashable).digest()
-        signature = hmac.new(
+        signature_digest = hmac.new(
             base64.b64decode(self.secret),
             message,
             hashlib.sha512,
-        )
+        ).digest()
```

Comment:
why not move this one too? Also why do we need to do the base64 decoding each time?